src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.MetadataDb.cs

```diff
@@ -148,7 +148,8 @@ internal MetadataDb(MetadataDb source, bool useArrayPools)
 
```

Comment:
Is the above constructor called anywhere?

```C#
internal MetadataDb(MetadataDb source, bool useArrayPools)
```

I don't see it being referenced (although I might not be looking everywhere I should be...)

Reply:
Looks like it may have gotten left behind when the feature that used it got cut.

Reply:
Ok, good to know I wasn't missing something. I was looking through when/why `_isLocked` gets set, and wanted to ensure all the places where it was `true` didn't result in MetadataDb getting disposed.